src/shared/component/SwaggerURLInput.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import './SwaggerURLInput.scss';
+
+class SwaggerURLInput extends React.Component {
+
+  onSubmit(e) {
+    e.preventDefault();
+
+    const swaggerURL = this.swaggerURLInput.value;
```

Comment:
It's not good to have properties defined like `this.swaggerURLInput`.
If it's not a prop and it stays inside the current component, it's probably a state. At the minimum, it should be defined in a constructor, because it's not clear where this is coming from.